### ISSUE TITLE
trigger one workflow from another workflow 

### ISSUE THREAD

**Issue by mmoya91** [temporalio/temporal#5372]
I was wondering if it was possible to trigger one workflow using another workflow? For instance: can workflow A trigger workflow B and can workflow B access outputs from workflow A's activity?

We want to build two different workflows for two different teams that share the same temporal server

**Comment by theleeeo** [comment]
Yes, it is possible to trigger workflows from within a workflow. Accessing activity results is slightly more difficult but still possible.
If you are wondering from a security perspective, you should definitely use two different servers to keep everything separate. If you only want the two teams to not accidentally interfere with each others activity- and workflow names using two namespaces will likely be enough.
